specs-go/chain/chainid.go

```diff
@@ -0,0 +1,46 @@
+package chain
+
+import "github.com/docker/distribution/digest"
+
+// ChainID takes a slice of digests and returns the ChainID corresponding to
+// the last entry. Typically, these are a list of layer DiffIDs, with the
```

Comment:
@wking Seriously, if you continue to split hairs on topics you have literally _no clue about_, I am going to block you. In fact, I remember explaining chain ids to you _months_ ago and having you walk away with no understanding in their use or calculation.

Now, you show up, quoting yourself, rather than the proposed explanation, as canonical?

> And the ChainID being returned here corresponds to the whole slice, not just the last entry.

These are the same thing.